docs/installation.rst

```diff
@@ -131,22 +128,22 @@ Once you have a copy of the source, you can install it with:
 .. _tarball: https://github.com/TissueImageAnalytics/tiatoolbox/tarball/master
 
 Using Docker
-============
+------------
 
 To run TIA toolbox in an isolated environment, use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our github repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_. Please report any issues related to the docker image in the repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_.
 
 After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desktop), you can use our TIA toolbox image in 3 different ways.
 
-Use the pre-built docker image
+Use the Pre-Built Docker Image
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-1. Pull the image from the Github Container Registry
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1. Pull the Image From Github Container Registry
```

Comment:
TIAToolbox provides pre-built docker containers which can be downloaded using the instructions below: